tensorbay/utility/file.py

```diff
@@ -16,6 +16,7 @@
 
 from _io import BufferedReader
 
+from tensorbay.client.requests import config
```

Comment:
this seems weird because module `utility` imports from `client`. looks like a cyclic import in module level.